### ISSUE TITLE
Danger deleted comments by other users on Github private repo

### ISSUE THREAD

**Issue by stevethomp** [danger/danger-js#552]
From a jenkins pull request job, Danger JS 3.4.3 deleted comments by other users. Logs show no comment should have been posted so it deleted the Danger comment, but also many from other users. We had been using previous versions and 3.3.2 didn't do this, although I'm not sure exactly which version could have introduced the issue.

09:10:50 + yarn add danger
09:10:50 yarn add v1.5.1
09:10:50 warning package.json: No license field
09:10:51 warning No license field
09:10:51 [1/4] Resolving packages...
09:10:53 [2/4] Fetching packages...
09:10:53 [3/4] Linking dependencies...
09:10:54 [4/4] Building fresh packages...
09:10:54 success Saved lockfile.
09:10:54 success Saved 1 new dependency.
09:10:54 info Direct dependencies
09:10:54 └─ danger@3.4.3
09:10:54 info All dependencies
09:10:54 └─ danger@3.4.3
09:10:54 warning No license field
09:10:54 Done in 3.86s.
09:10:54 + yarn danger ci
09:10:55 yarn run v1.5.1
09:10:55 warning package.json: No license field
09:10:55 $ /Users/user/jenkins/workspace/kinzoo-ios-pull-requests/node_modules/.bin/danger ci
09:11:00 No issues or messages were sent. Removing any existing messages.
09:11:06 Done in 10.79s.

![](https://user-images.githubusercontent.com/348937/38205367-30ea3778-365b-11e8-999f-1073cd5eb8bb.png)

**Comment by sunshinejr** [comment]
Hey @stevethomp, thanks for the issue. Sorry that it happened to you, it is for sure a regression from the inline mode we introduced in Danger JS 3.4.0+. Can you tell me if these comments were inline comments? Or just the normal, regular comments?

**Comment by stevethomp** [comment]
Yup, all the deleted comments were inline comments, there was one previous normal comment but Danger didn't touch that one.

**Comment by orta** [comment]
Thanks for trying new versions! (And then reporting the 🐛s )
